src/mame/subsino/subsino.cpp

```diff
@@ -3279,7 +3322,44 @@ ROM_START( newhunter )
 	ROM_LOAD( "ht-4.bin",     0x00345, 0x00117, CRC(f4f78925) SHA1(738281c05c0f51c0edcf65bacc18ebc87c2408c1) )
 ROM_END
 
+/*  The PCB have five 8-dipswitches bank sockets, but four of them were unpopulated (only one was present on the PCB).
+    The socket for the reset switch was also unpopulated on the PCB, and there wasn't memory backed RAM on the PCB
+    (present on most games on this hardware). */
```

Comment:
“The PCB had five positions for banks of eight DIP switches, but for of them were unpopulated…”
“…there was no battery-backed RAM…”

(Assuming that’s what you meant – “memory backed RAM” doesn’t make a lot of sense.)

Reply:
Fixed, thanks  :D